Gemfile

```diff
@@ -10,3 +10,5 @@ gem 'rubocop'
 gem 'timecop'
 gem 'byebug'
 gem 'codecov', require: false, group: :test
+gem 'ruby-prof', require: false, group: :test
```

Comment:
Gemfile in the gem repo is only used for dev and testing. It's totally separate from gemspec dependencies. I don't think using groups makes a lot of sense here.